src/util/domains.py

```diff
@@ -92,6 +93,8 @@ def is_leaf(leaf):
     AND_OPERATOR = expression.AND_OPERATOR
     OR_OPERATOR = expression.OR_OPERATOR
     DOMAIN_OPERATORS = expression.DOMAIN_OPERATORS
+    if not on_CI():
```

Comment:
Why only on CI?

Reply:
It should still be shown for non-upgrade builds, as `odoo.upgrade.util` is not imported.

Reply:
Yes but would those builds cover the _same_ lines... :/
Anyway I'll remove it.

Reply:
Done